src/devices/configured-device-card.ts

```diff
@@ -44,19 +44,20 @@ class ESPHomeConfiguredDeviceCard extends LitElement {
                   <a href=${`http://${this.entry.address}`} target="_blank">
                     <mwc-icon>launch</mwc-icon>
                   </a>
-                  <paper-tooltip>Open Node Web Server Interface</paper-tooltip>
+                  <paper-tooltip>
+                    Open device web server interface
+                  </paper-tooltip>
                 </div>
               `
             : ""}
           ${this.device.deployed_version != this.device.current_version
             ? html`
                 <div class="tooltip-container">
                   <mwc-icon class="update-available">system_update</mwc-icon>
-                  <paper-tooltip
-                    >Update Available: ${this.device.deployed_version}
-                    ${UPDATE_TO_ICON}
-                    ${this.device.current_version}</paper-tooltip
-                  >
+                  <paper-tooltip>
+                    Update Available: ${this.device.deployed_version}
```

Comment:
Should there be a `<br>` here ?

Reply:
No, the newline is just from prettier reformatting,

It should display like this (as it was before too):

Update Available: {old_ver} :arrow_right: {new_ver}